webpack.config.js

```diff
@@ -21,11 +21,10 @@ const availableStagingLocales = process.env.L10N
   : JSON.stringify(undefined);
 
 const config = {
-  entry: ['./src/index'],
+  entry: './src/index',
   output: {
     path: path.join(__dirname, 'dist'),
-    filename: '[hash].bundle.js',
-    chunkFilename: '[id].[hash].bundle.js',
+    filename: '[name].[hash].bundle.js',
```

Comment:
I added the [name] part because this was needed with a previous iteration of the patch where I added another chunk. I thought it would be good to keep in case we have this same needs in the future, to prevent surprises (it took me a few minutes to realize that the chunks were being overwritten without a message -- or I missed the message).